src/components/CurrentWeather.vue

```diff
@@ -37,11 +37,12 @@ export default defineComponent({
 		const city = ref('');
```

Comment:
I can't commen't evewhere in this file so leaving this comment in first available line. 
If you want your vue to be modern and respect recommendations from creators you should not use `defineComponent` any longer. `defineComponent` is a [helper](https://vuejs.org/api/general.html#definecomponent) with typing support. It was used in early stages of migration from vue2 to vue3. Was very helpful when big project written in vue 2 planned migration. Thanks to composition api plugin devs could start rewrite code to vue3-like syntax but not yet changing vue version. Most recommended way of writting script part of Single File Component in modern vue3 is to use `script setup` syntax. Please follow with [docs](https://vuejs.org/guide/scaling-up/sfc#introduction) to provide desired api to your vue app.